mobify/platform-scaffold
# Created a PrivacyPage component as part of a training exercise.

(brief description / PR title)

 **JIRA**: none
 **Linked PRs**: none

## Changes
- Added PrivacyPage container
- Added a new function to parser-utils called getTextArrayFrom to grab an array of text elements from the DOM

## How to test-drive this PR
- Run `npm run dev`
- Click on Privacy and Cookie Policy link
- Page should load via PWA instead of desktop site when navigating away and coming back
- Intro paragraph should be extracted from desktop site
- List of cookies should be extracted from desktop site
- Look for the paragraph and cookie list in the app state using React dev tools
